{{cookiecutter.project_slug}}/docker-compose.yml

```diff
@@ -1,25 +1,32 @@
-version: '2'
+version: '3'
 services:
 
   db:
     image: mdillon/postgis
-    volumes:
-      - pgdata:/var/lib/postgresql/data/pgdata
```

Comment:
Deze is niet meer nodig?

Reply:
Zo'n named volume is zodat je database data intact blijft als je docker-compose down doet. Zelf doe ik geen down maar stop, zodat de data wel intact blijft.
Maar als mensen dat gewend zijn wil ik de named volume erbij zetten.